cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -63,6 +63,8 @@ spec:
                   type: boolean
                 continueUpgradeAfterChecksEvenIfNotHealthy:
                   type: boolean
+                waitTimeoutForHealthyOSD:
```

Comment:
Why not change this to either
- a string which can take `1m` or similar values, OR
- rename this to `waitTimeoutForHealthyOSDInMinutes`
so that users don't have to look at docs to understand units?

Reply:
using `waitTimeoutForHealthyOSDInMinutes`.